terminus-ui/tooltip/testing/src/test-components.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Component,
+  NgModule,
+  ViewChild,
+} from '@angular/core';
+
+import {
+  TsTooltipComponent,
+  TsTooltipModule,
+} from '../../src/tooltip.module';
```

Comment:
This import should be like an external consumer: `@terminus/ui/tooltip`

Reply:
Yes, this was supposed to be temporary